Control/Monad/Trans/Random/Lazy.hs

```diff
@@ -283,6 +283,9 @@ instance (Monad m, RandomGen g) => StatefulGen (RandGen g) (RandT g m) where
   uniformWord32 = applyRandT genWord32
   uniformWord64 = applyRandT genWord64
   uniformShortByteString n = applyRandT (genShortByteString n)
+#if MIN_VERSION_random(1,3,0)
+  uniformByteArrayM pinned sz = applyRandT $ uniformByteArray pinned sz
+#endif
```

Comment:
I'd actually recommend switching to default implementation for `uniformShortByteString`, since it is now deprecated (it uses the underneath `uniformByteArrayM` for default implementation from 1.3.0 onwards) and will be removed in some future . This will improve future compatibility.
```suggestion
#if MIN_VERSION_random(1,3,0)
  uniformByteArrayM pinned sz = applyRandT $ uniformByteArray pinned sz
#else
  uniformShortByteString n = applyRandT (genShortByteString n)
#endif
```